packages/web-react/src/components/FileUploader/README.md

```diff
@@ -413,22 +413,20 @@ and [escape hatches][readme-escape-hatches].
-| `label`           | `string`                             | —        | ✔        | File name                                                                   |
-| `name`            | `string`                             | —        | ✔        | Input field name                                                            |
-| `onDismiss`       | `(key: string) => FileQueueMapType`  | —        | ✔        | Callback to delete an item from the queue                                   |
-| `onEdit`          | `(event: Event, file: File) => void` | —        | ✕        | Show and add function to edit button                                        |
-| `onError`         | `FileUploaderErrorCallbackType`      | —        | ✕        | Callback on error condition                                                 |
-| `removeText`      | `string`                             | `Remove` | ✕        | Dismiss button label                                                        |
+| Name              | Type                                 | Default  | Required | Description                                           |
+| ----------------- | ------------------------------------ | -------- | -------- | ----------------------------------------------------- |
+| `editText`        | `string`                             | `Edit`   | ✕        | Edit button label                                     |
+| `file`            | `File`                               | —        | ✔        | Attachment file object                                |
+| `hasImagePreview` | `bool`                               | false    | ✕        | Show image preview                                    |
+| `iconName`        | `string`                             | `file`   | ✕        | Icon shown along the file                             |
+| `id`              | `string`                             | —        | ✔        | FileUploaderAttachment id                             |
+| `imageObjectFit`  | [`cover` \| `contain`]               | `cover`  | ✕        | Defines FileUploaderAttachment image fit in container |
+| `label`           | `string`                             | —        | ✔        | File name                                             |
+| `name`            | `string`                             | —        | ✔        | Input field name                                      |
+| `onDismiss`       | `(key: string) => FileQueueMapType`  | —        | ✔        | Callback to delete an item from the queue             |
+| `onEdit`          | `(event: Event, file: File) => void` | —        | ✕        | Show and add function to edit button                  |
+| `onError`         | `FileUploaderErrorCallbackType`      | —        | ✕        | Callback on error condition                           |
+| `removeText`      | `string`                             | `Remove` | ✕        | Dismiss button label                                  |
```

Comment:
I was wondering if the description could be improved, it's rather a "remove" button than a "dismiss" button (it removes a file from the upload queue).

```suggestion
| `removeText`      | `string`                             | `Remove` | ✕        | Remove button label                                  |
```

Reply:
updated 👍